iaso/api/org_unit_tree/views.py

```diff
@@ -60,7 +60,8 @@ def get_queryset(self):
         display_root_level = not parent_id
 
         if display_root_level and self.action == "list":
-            if user.is_authenticated and user.iaso_profile.org_units.exists():
+            force_full_tree = force_full_tree or user.is_superuser
```

Comment:
front-end is never using this param, i don't understand why we have it

Reply:
This is currently unused but it was asked by @madewulf and specified in the design doc here https://docs.google.com/document/d/1LczWRYbDCJtbxG2fd_c1WtQ4DmNclRG2ixXnr_C58wo

I thought about removing it because I misimplemented it at first, but now it works and it's well tested.

I can remove it if everyone is OK, or we could use it in the front-end for a new feature.